js/gratipay/team.js

```diff
@@ -61,43 +56,50 @@ Gratipay.team = (function() {
-                    , ['td', {'class': 'figure last_week'}, num(member.last_week)]
-                    , ['td', {'class': 'figure take ' + increase}, drawMemberTake(member)]
-                    , ['td', {'class': 'figure balance'}, num(member.balance)]
-                    , ['td', {'class': 'figure percentage'}, perc(member.percentage)]
-                     ]
-                ));
-            else if (nmembers > 0)
-                rows.push(Gratipay.jsonml(
-                    [ 'tr'
-                    , ['td']
-                    , ['td']
-                    , ['td']
-                    , ['td', {'class': 'figure take'}, num(member.take)]
-                    , ['td', {'class': 'figure balance'}, num(member.balance)]
-                    , ['td', {'class': 'figure percentage'}, perc(member.percentage)]
-                     ]
-                ));
+            rows.push(Gratipay.jsonml(
+                [ 'tr'
+                , ['td', {'class': 'n'}, (i === nmembers ? '' : nmembers - i)]
+                , ['td', ['a', {'href': '/~'+member.username+'/'}, member.username]]
+                , ['td', {'class': 'figure last_week'}, num(member.last_week)]
+                , ['td', {'class': 'figure take ' + increase}, drawMemberTake(member)]
+                , ['td', {'class': 'figure balance'}, num(member.balance)]
+                , ['td', {'class': 'figure percentage'}, perc(member.percentage)]
+                 ]
+            ));
+        }
+
+        if (nmembers === 0) {
+            rows.push(Gratipay.jsonml(
+                ['tr', ['td', {'colspan': '6', 'class': 'no-members'}, "No members"]]
+            ));
+        } else {
+            rows.push(Gratipay.jsonml(
+                [ 'tr'
+                , {'class': 'totals'}
+                , ['td']
+                , ['td']
+                , ['td']
+                , ['td', {'class': 'figure take'}, num(available - member.balance)]
+                , ['td', {'class': 'figure balance'}, num(member.balance)]
+                , ['td', {'class': 'figure percentage'}, perc(member.balance / available)]
+                 ]
+            ));
         }
-        $('#team-members').html(rows);
-        $('#take').submit(doTake);
-        $('#take input').focus().keyup(maybeCancelTake);
-        $('#team-members .remove').click(remove);
+
+        $t('.team-members').html(rows);
+        $t('.team-members .edit').submit(doTake);
+        $t('.team-members .edit input').focus().keyup(maybeCancelTake);
+        $t('.team-members .remove').click(remove);
     }
 
 
     // Add
     // ===
 
     function lookup() {
-        var query = $('#query').val();
+        var query = $t('.query').val();
         if (query === '')
-            $('#lookup-results').empty();
+            $t('.lookup-results').empty();
         else
             jQuery.get("/lookup.json", {query: query}).success(drawLookupResults);
```

Comment:
As @nashe points out in https://github.com/gratipay/gratipay.com/pull/4077/files#r73421480, this endpoint returns users that don't pass the validation for new team members. In e80433913ec6030987c9a9007d16050432fb83d6 we have a start on wiring up proper error messaging, but I think it'd be better to go the alternate route that @nashe suggests, and not include these non-allowed users in the first place. The best would probably be to include them in `lookup.json` results, but with a flag so that we can show in the UI that "this person exists—you got their username right—but they're not eligible for such-and-such reason."